index.js

```diff
@@ -0,0 +1,36 @@
+const $taskInput = document.getElementById('taskInput');
+const $taskList = document.getElementById('taskList');
```

Comment:
DOM에 접근을 할 때 마다 document.getElementById 메소드를 사용하면 많은 양의 코드를 작성하게 될탠데요. `$()`과 같은 DOM에 접근할 수 있는 유틸 함수를 한번 만들어보면 어떨까요?

키워드
- 유틸 함수
- 모듈
- import, export

```suggestion
const $taskInput = $('#taskInput');
const $taskList = $('#taskList');
```

Reply:
dom에 접근할 때, class가 아닌 id로 접근할 계획이므로 

function $(selector) {
  return document.getElementById(selector);
}

이렇게 변경해봤습니닷!